rasa/core/training/interactive.py

```diff
@@ -49,7 +49,10 @@
 )
 from rasa.core import run, utils
 import rasa.core.train
-from rasa.core.constants import DEFAULT_SERVER_FORMAT, DEFAULT_SERVER_PORT
+from rasa.core.constants import (
+    DEFAULT_INTERACTIVE_SERVER_URL,
```

Comment:
Do we actually need this? I think it would be fine to use the new `DEFAULT_SERVER_FORMAT` everywhere in this file as `0.0.0.0` should be ok.